.github/ISSUE_TEMPLATE/feature.yml

```
@@ -0,0 +1,249 @@
+        - JWT token management with 24-hour expiration
+        - Automatic token refresh
+        - Secure logout with token invalidation
+
+        Non-Functional Requirements:
+        - Response time: < 500ms for authentication
+        - Support for 1000+ concurrent users
+        - WCAG 2.1 AA compliance
+        - Mobile responsive design
+      value: |
+        ### Functional Requirements:
+        - [ ] 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)